deploy/examples/example-custom-build-images.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoApp
+metadata:
+  name: example-quarkus
+spec:
+  build:
+    #env:
+    # this value is just an example, set yours (optional)
+    #- name: MAVEN_MIRROR_URL
+    #  value: "http://nexus3.kogito-tools.svc.cluster.local:8081/repository/maven-public/"
+    gitSource:
+      contextDir: jbpm-quarkus-example
+      uri: 'https://github.com/kiegroup/kogito-examples'
+      reference: 0.7.0-rc4
+    # images that don't have a custom tag, will use this version
+    imageVersion: 0.7.0-rc4
+    imageRuntime:
+      # the image runtime build will use this image
+      imageStreamTag: quay.io/ricardozanini/kogito-quarkus-jvm-ubi8:0.7.0-rc1
```

Comment:
maybe use the kiegroup tag 0.7.7-rc4?

Reply:
This is to show that we can have a custom tag, thus the namespace naming (and with an actual tag that exists).